setup.cfg

```diff
@@ -1,3 +1,48 @@
+[metadata]
+name = napari-arboretum
+author = Alan R. Lowe
+author_email = a.lowe@ucl.ac.uk
+license = LICENCE.md
+url = https://github.com/quantumjot/arboretum
+description = Track graph and lineage tree visualization with napari
+long_description = file: README.md
+long_description_content_type = text/markdown
+python_requires = >=3.8
```

Comment:
Is there a way of having `>=3.8, 3.11` in case `3.11` has some major change as don't officially support it?